### PR TITLE
Add Chatters beta Helix endpoint

### PR DESCRIPTION
**Pull request by zunderscore**
Type: Feature

Fixes: #424

Add Chatters beta Helix API endpoint

### CODE REVIEW

File: packages/api/src/api/helix/chat/HelixChatApi.ts [hunk 1]
```diff
@@ -125,6 +128,36 @@ export type HelixChatUserColor =
  */
 @rtfm('api', 'HelixChatApi')
 export class HelixChatApi extends BaseApi {
+	/**
+	 * Gets the list of users that are connected to the broadcaster’s chat session.
+	 *
+	 * @beta
+	 * @param broadcasterId The ID of the broadcaster in whose list of chatters you want to get.
+	 * @param moderatorId The ID of the broadcaster or one of the broadcaster’s moderators.
```
Comment (d-fischer) [inline packages/api/src/api/helix/chat/HelixChatApi.ts:136]: "The ID of" is wrong here, as you can always also pass user objects. Also I think there's a stray "in" in the first one.

**Review by d-fischer — CHANGES_REQUESTED** [review]

File: packages/api/src/api/helix/chat/HelixChatApi.ts (hunk 1 above)
Comment (zunderscore) [reply packages/api/src/api/helix/chat/HelixChatApi.ts]: "The ID of" is used in quite a few other places (see Helix moderation API), but I updated the verbiage and parameter names here.

**Comment by zunderscore** [comment]
Changes made

**Review by d-fischer — APPROVED** [review]

File: packages/api/src/api/helix/chat/HelixChatChatter.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { Enumerable } from '@d-fischer/shared-utils/lib';
```
Comment (d-fischer) [inline packages/api/src/api/helix/chat/HelixChatChatter.ts:1]: Fails at runtime due to `exports` definition.

**Review by d-fischer — CHANGES_REQUESTED** [review]

File: packages/api/src/api/helix/chat/HelixChatChatter.ts (hunk 2 above)
Comment (zunderscore) [reply packages/api/src/api/helix/chat/HelixChatChatter.ts]: Autoreference strikes again. Fixed.

**Review by d-fischer — APPROVED** [review]

**Pull request merged**